content/en/observability_pipelines/best_practices_for_scaling_observability_pipelines.md

```diff
@@ -51,7 +51,7 @@ Observability Pipelines Worker runs on modern CPU architectures. X86_64 architec
 
 ### Memory sizing
 
-Due to Observability Pipelines Worker's affine type system, memory is rarely constrained for Observability Pipelines Worker workloads. Therefore, Datadog recommends ≥2 GiB of memory per vCPU minimum. Memory usage increases with the number of sinks due to the in-memory buffering and batching. If you have a lot of sinks, consider increasing the memory or switching to disk buffers.
+Due to Observability Pipelines Worker's affine type system, memory is rarely constrained for Observability Pipelines Worker workloads. Therefore, Datadog recommends ≥2 GiB of memory per vCPU minimum. Memory usage increases with the number of sinks due to the in-memory buffering and batching. If you have a lot of sinks, consider increasing the memory.
 
 ### Disk sizing
```

Comment:
@ckelner do we want to remove this "Disk Sizing" section as well?

Reply:
Per https://vector.dev/docs/setup/going-to-prod/sizing/#disks it states:
> Sizing disks is only relevant if you’re using Vector’s disk buffers.

But I feel like we should have some kind of guidance here, even w/o disk buffers.
Let me ask the OP backroom, I have a thread on this topic.

Reply:
@maycmlee I think for "Disk Sizing" we just say "30GB minimum" and leave it at that.

Reply:
@ckelner could you let me know if the update below is what you are suggesting? I'm assuming the example caclulation is still correct?

> For example, an 8 vCPU machine would require 288 GiB of disk space (10 MiB * 60 seconds * 60 minutes * 8 vCPUs).

Reply:
Chatted on zoom. Remove the entire section and just add the minimum required sentence.